synapse/storage/data_stores/main/purge_events.py

```diff
@@ -361,7 +361,6 @@ def _purge_room_txn(self, txn, room_id):
             "event_push_summary",
             "pusher_throttle",
             "group_summary_rooms",
-            "local_invites",
```

Comment:
Would we still want to potentially purge the table until it is fully removed from the database? I doubt it matters much either way, but...

Reply:
I'm minded to say "no". Say we leave this line here for now and drop the table in Synapse 1.17.0: that will mean that anybody trying to roll back from 1.17.0 to 1.16.0 will end up with broken purges.

OTOH leaving it here seems fairly harmless, especially if we actually drop the table soon.

Reply:
I can see it going either way. I guess my thought was to think about *why* an admin might be purging a room and if it could leave them liable to have information in the `local_invites` table still.

I think removing it from the purges is the simplest way for us to handle rollbacks, however. I'm convinced!